src/ui_gtk3.hh

```diff
@@ -8297,6 +8297,22 @@ int _gtk_loop(void *p)
         gtk_window_set_resizable(GTK_WINDOW(info_dialog), true);
         gtk_window_set_default_size(GTK_WINDOW(info_dialog), 425, 400);
 
+        gtk_window_set_decorated(GTK_WINDOW(info_dialog), FALSE);
+
+        GtkCssProvider *provider = gtk_css_provider_new();
+        gtk_css_provider_load_from_data(provider,
```

Comment:
Styling just one dialog like this makes everything else look inconsistent. And I've said this before but we're moving away from GTK3 and it's really not a good idea to spend any more time on the GTK dialog code than is absolutely necessary.